examples/renameimage.py

```diff
@@ -36,48 +104,109 @@
         logging.error(msg)
         sys.exit(-1)
 
+    old_name = sys.argv[1]
+    new_name = sys.argv[2]
+
     glancesync = GlanceSync()
     glancesync.init_logs()
+    master = glancesync.master_region
+
+    master_images = glancesync.get_images_region(master)
+
+    found_with_old_name = None
+    found_with_new_name = None
+    for image in master_images:
+        if image.name == old_name:
+            if found_with_old_name:
+                msg = 'There are more than an image with the old name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                found_with_old_name = image
+        elif image.name == new_name:
+            if found_with_new_name:
+                msg = 'There are more than an image with the new name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                found_with_new_name = image
+
+    if found_with_old_name and found_with_new_name:
+        msg = 'There is an image with the same old and new name in the master'\
+            ' region'
+        logging.error(msg)
+        sys.exit(-1)
+
+    if found_with_new_name:
+        master_image = found_with_new_name
+    elif found_with_old_name:
+        master_image = found_with_old_name
+        # Rename
+        master_image.name = sys.argv[2]
+        glancesync.update_metadata_image(master, master_image)
+        print('Renamed image in master region')
+    else:
+        msg = 'Neither an image with the old name nor a one with the new name'\
+            ' exist in the master region'
+        logging.error(msg)
+        sys.exit(-1)
+
     if len(sys.argv) > 3:
         regions = sys.argv[3:]
     else:
-        regions = glancesync.get_regions(False)
+        regions = glancesync.get_regions()
 
     for region in regions:
-        print region + ' ',
+        sys.stdout.write(region + ' ')
```

Comment:
Because the region name is printed before making the call. The call may last a while or even fail. This is similar to the init start scripts, where a service name is printed before starting and then OK or FAIL